### PR TITLE
cascade container defs on task def update/delete

### PR DESCRIPTION
**Pull request by depombo**

### CODE REVIEW

File: src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts [hunk 1]
```diff
@@ -31,7 +31,11 @@ export class ContainerDefinition {
   @Column()
   name: string;
 
-  @ManyToOne(() => TaskDefinition)
+  @ManyToOne(() => TaskDefinition, {
+    // if the parent task def is deleted, also delete this container def
```
Comment (dfellis) [inline src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts:35]: Comment seems slightly out of sync with the code here?

**Review by dfellis — APPROVED** [review]
Minor comment, but LGTM assuming the end-to-end test works, too :)

File: src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts (hunk 1 above)
Comment (depombo) [reply src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts:35]: typeorm is a bit funny in that you define this in the child as opposed to the parent which took me some time to realize :/

Comment (dfellis) [reply src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts:35]: So I mean you're also doing it if the parent task definition is updated, afaict?

Comment (depombo) [reply src/modules/aws_ecs_fargate@0.0.1/entity/container_definition.ts:35]: ooh yes, that part of the comment is incomplete. I see what you mean, will update in the next PR

**Pull request merged**